packages/gatsby-theme-docs/src/layouts/internals/footer.js

```diff
@@ -121,17 +121,6 @@ const AlignedRight = styled.div`
 const LayoutFooter = () => {
   const data = useStaticQuery(graphql`
     query GetFooterLinks {
-      allTopMenuYaml {
-        nodes {
-          id
-          menuTitle
-          items {
-            label
-            href
-            beta
-          }
-        }
-      }
```

Comment:
This is also being changed in the footer PR. So it would be best if we merge this PR first because it's probably easier to resolve the conflicts in my PR.